package.json

```
@@ -77,7 +77,7 @@
   },
   "devDependencies": {
     "cross-env": "^7.0.3",
-    "eslint": "~7.18.0",
+    "eslint": "~7.19.0",
```

This should be done once <https://github.com/eslint/eslint/issues/14058> is resolved:

```suggestion
    "eslint": "^7.19.0",
```